experimental/StandardFiniteFields/test/runtests.jl

```diff
@@ -0,0 +1,44 @@
+using Test
+using Oscar
+GAP.Packages.load("StandardFF")
+
+function MyPoly(p,n)
+    return defining_polynomial(standard_finite_field(p,n))
+end
+
+function GAPPoly(p,n)
+    poly = GAP.evalstr( "StandardFiniteField(" * string(p) * ", " * string(n) * ")!.DefiningPolynomial!.CoefficientsOfUnivariatePolynomial")
+    poly = polynomial(GF(p), map(x -> GF(p)(x), GAP.gap_to_julia(Vector{GAP.FFE}, poly)))
+end
+
+small_test_primes = ZZ.([3, 5, 7])
+medium_test_primes = ZZ.([3433, 4073])
```

Comment:
These two don't seem to be used. However the tests later on reference `small_primes` which in turn doesn't seem to be defined anywhere?

Reply:
thanks, this was an oversight about `small_primes` vs `small_test_primes`. I decided that testing with small primes should be sufficient for now, so I commented out the `medium_test_primes`.